### ISSUE TITLE
Other files have been modified; changes not deployed

### ISSUE THREAD

**Issue by nosovk** [helen/action-wordpress-plugin-deploy#88]
After creating new tag got an error on deploy.
`Other files have been modified; changes not deployed`
**Describe the bug**
[link to logs](https://github.com/MailCheck-co/mailcheck-wordpress/runs/5613851306?check_suite_focus=true)

It seems that it worked before, but have broken some moment

**Comment by nosovk** [comment]
```
svn: E155011: Commit failed (details follow):
svn: E155011: Directory '/home/runner/svn-mailcheck-co/tags/0.5' is out of date
svn: E175014: Precondition on '/!svn/rvr/2696510/mailcheck-co/trunk' failed
```
it seems that svn has incorrect state

**Comment by emtiazzahid** [comment]
Hi, did you found any solution for that?

**Comment by nosovk** [comment]
nope, have no solution

**Comment by helen** [comment]
Hi - the logs you linked indicate there were modifications in `trunk/mailcheck-co.php`, meaning in your `master` branch at the time there were changes beyond asset files since the last sync with `trunk` on SVN so the action doesn't run out of an abundance of caution. Not sure where the other errors you pasted happened, so a link to those may help, but perhaps it's resolved itself in the meantime?

**Comment by nosovk** [comment]
Hm, but if I use only that action to communicate with SVN, how master could be beyond? I even have no SVN client to modify smith avoiding action launch.

**Comment by richard-muvirimi** [comment]
This is addressed by #96

**Comment by nosovk** [comment]
You mean that it should be fixed now?

**Comment by richard-muvirimi** [comment]
@nosovk the fix has been merged into the development branch, but not sure when the next release would be.

**Comment by dillix** [comment]
@richard-muvirimi we tried latest **stable** version 2.2.1 and error:

```
svn: E155011: Commit failed (details follow):
[961](https://github.com/uploadcare/uploadcare-wordpress/actions/runs/3713205639/jobs/6295812414#step:6:963)
svn: E155011: Directory '/home/runner/svn-uploadcare/tags/3.0.9' is out of date
[962](https://github.com/uploadcare/uploadcare-wordpress/actions/runs/3713205639/jobs/6295812414#step:6:964)
svn: E175014: Precondition on '/!svn/rvr/2835058/uploadcare/trunk' failed
[963](https://github.com/uploadcare/uploadcare-wordpress/actions/runs/3713205639/jobs/6295812414#step:6:965)
Error: Process completed with exit code 1.
```

**Comment by jeffpaul** [comment]
Closed by #96
